package.json

```diff
@@ -4,8 +4,22 @@
   "description": "js lab first milestone project called \"Blog API\"",
   "main": "server.js",
   "scripts": {
+    "start": "node dist/index.js",
+    "dev": "nodemon src/index.ts",
+    "build": "tsc",
     "test": "jest"
   },
   "author": "bek-shoyatbek",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "@types/express": "^5.0.0",
```

Comment:
I would suggest to install types and nodemon as dev dependency because they will be used for development only. Not for final product

Reply:
Thanks , I'll